### DISCUSSION TITLE
Bug with "Create List" and emojis

### DISCUSSION THREAD

**Discussion by yagueto** [community/community#9259] [Other Feature Feedback, Questions, & Ideas]
I was trying the new lists but, when I tried to add an emoji, the popup with suggestions had an issue with the icons size (see screenshot below)

![imagen](https://user-images.githubusercontent.com/48566793/147285159-7d7bd0c0-7ce6-4f8d-9f68-2d32c5892345.png)

**Comment by brianlovin** [comment]
Thanks for the heads up @yagueto — I'll pass this along to our team to investigate what might be going on here!

**Comment by cheshire137** [comment]
Thank you for the report! I was unable to reproduce this in Windows 10 in Firefox, Edge, or Chrome. I wonder if there's some other difference in our setups that's making the emoji sizes much bigger for you.

**Reply by yagueto** [reply]
I've just checked it again and it looks like it doesn’t happen anymore 😬. For the records, I was using Firefox in Arch.

**Reply by cheshire137** [reply]
Good deal, I'm glad it's working now!

**Reply by Satrio21** [reply]
:))

**Comment by rothgar** [comment]
I'm using Chrome 96.0.4664.110 on Fedora 35 and see this with emoji.
![image](https://user-images.githubusercontent.com/371796/148668146-3f9afdcc-c164-4040-b317-0a584bac72fe.png)

**Reply by brianlovin** [reply]
@rothgar are you seeing those incorrectly-sized emoji in other places on GitHub? Like when typing a comment, or updating the status on your profile?

**Reply by rothgar** [reply]
No, this was the first time I've seen that

**Comment by Cesar556** [comment]
> Thanks for the heads up @yagueto — I'll pass this along to our team to investigate what might be going on here!

- [ ] []()_@cheshire137 _

**Comment by farazahmad96** [comment]
when I create list directly from start assigning, it stuck on creating list and on refresh it has created list.

**Comment by rainbow1016** [comment]
very good question!

**Comment by 98477772** [comment]
_[@~~~~](https://github.com/karlomikus/bar-assistant/actions/runs/5904326799)_
